docs/docs/dev/security/certificates.md

```diff
@@ -122,6 +122,46 @@ You can also provide the CA via environment variable `KUMA_CONTROL_PLANE_CA_CERT
 
 See [Data plane proxy authentication](./dp-auth.md) and [Zone Ingress authentication](./zone-ingress-auth.md).
 
+## Prometheus to control plane communication
```

Comment:
Just a note, for 2.0, this doc was moved to `src/features/zero-trust/certificates.md`